### ISSUE TITLE
Win32/Uwasson.A!ml

### ISSUE THREAD

**Issue by GeorgesOatesLarsen** [nim-lang/nimble#930]
nimgrep.exe,, testament.exe, and vccexe.exe are triggering Windows Defender to warn about Win32/Uwasson.A!ml

Is anyone able to elaborate on this? Is this a false positive?

**Comment by dom96** [comment]
Yep, this is a false positive. Lot's of people experiencing this, unfortunately not much we can do except complain to the AV vendors :/ https://forum.nim-lang.org/t/8196
